scripts/mirror.js

```diff
@@ -220,70 +218,62 @@ const bumpChromeAndroid = (originalData, sourceData, source) => {
 
 /**
  * @param {SupportStatement} originalData
- * @param {SupportStatement} sourceData
- * @param {string} source
+ * @param {SupportStatement} chromeData
+ * @param {SupportStatement} ieData
  * @returns {SupportStatement}
  */
-const bumpEdge = (originalData, sourceData, source) => {
-  let newData = {};
+const bumpEdge = (originalData, chromeData, ieData) => {
+  if (Array.isArray(originalData)) {
+    return originalData.map(d => bumpEdge(d, chromeData, ieData));
+  }
 
-  if (source == 'ie') {
-    if (sourceData.version_removed && sourceData.version_removed !== null) {
-      newData.version_added = false;
-    } else if (sourceData.version_added !== null) {
-      newData.version_added = sourceData.version_added ? '12' : null;
-    }
+  let newData = copyStatement(originalData);
 
-    if (sourceData.notes) {
-      newData.notes = updateNotes(
-        sourceData.notes,
-        /Internet Explorer/g,
-        'Edge',
-      );
+  if (ieData) {
+    if (ieData.version_removed || ieData.version_added === false) {
+      newData.version_added = false;
+    } else if (ieData.version_added) {
+      newData.version_added = ieData.version_added ? '12' : null;
```

Comment:
```suggestion
      newData.version_added = '12';
```

`ieData.version_added` is always truthy here.